core/vidl/vidl_ffmpeg_convert.cxx

```diff
@@ -27,9 +28,9 @@ extern "C" {
 }
 #endif
 
-
 //--------------------------------------------------------------------------------
 
+#if LIBAVCODEC_VERSION_MAJOR < 56
```

Comment:
Used typedef and macro definition to map the old types and values to the new ones.  Keeping the new ones as untouched, assuming this is the direction to go.   Code is pushed.